odabot/cli.py

```diff
@@ -250,28 +287,47 @@ def update_workflow(last_commit, last_commit_created_at, project):
                     ("Dear MMODA Workflow Developer\n\n"
                         "ODA-Workflow-Bot just deployed your workflow, and passed basic validation.\n"
                         "Please find below some details on the inferred workflow properties, "
-                        "and check if the parameters and the default outputs are interpretted as you intended them to beЖ.\n\n"
+                        "and check if the parameters and the default outputs are interpretted as you intended them to be.\n\n"
                         f"{json.dumps(deployment_info, indent=4)}\n\n"
                         "You can now try using accessing your workflow, see https://odahub.io/docs/guide-development/#optional-try-a-test-service\n\n"
                         "\n\nSincerely, ODA Bot"
                         ))
 
     return deployed_workflows            
-       
 
 
 @cli.command()
 @click.option("--dry-run", is_flag=True)
 @click.option("--loop", default=0)
 @click.option("--force", is_flag=True)
 @click.option("--pattern", default=".*")
-def update_workflows(dry_run, force, loop, pattern):
-    # TODO: config
+@click.pass_obj
+def update_workflows(obj, dry_run, force, loop, pattern):
+    if obj['settings'].get('nb2workflow.kb.type', 'odakb') == 'file':
+        from odabot.simplekb import TurtleFileGraph
+        odakb_sparql = TurtleFileGraph(obj['settings'].get('nb2workflow.kb.path'))
+    else:
+        import odakb
+        odakb_sparql = odakb.sparql    
+    
+    k8s_namespace = obj['settings'].get('nb2workflow.k8s_namespace', 'oda-staging')
+    dispatcher_url = obj['settings'].get('nb2workflow.dispatcher.url', 
+                                         "https://dispatcher-staging.obsuks1.unige.ch")
+    dispatcher_deployment = obj['settings'].get('nb2workflow.dispatcher.deployment', 
+                                                "oda-dispatcher")
+    container_registry = obj['settings'].get('nb2workflow.registry', 'odahub')
+    
+    frontend_instruments_dir = obj['settings'].get('nb2workflow.frontend.instruments_dir', None)
+    frontend_deployment = obj['settings'].get('nb2workflow.frontend.deployment', None)
+    
+    build_engine = obj['settings'].get('nb2workflow.build_engine', 'docker')
 
     while True:
         try:
             try:
-                oda_bot_runtime = yaml.safe_load(open('oda-bot-runtime-workflows.yaml'))
+                oda_bot_runtime = yaml.safe_load(open('oda-bot-runtime-workflows.yaml')) 
+                # TODO: Probably better to get this info from KG? 
```

Comment:
this is particular site state. it might be in a KG, but maybe it's better if it's in a local site KG.

Reply:
Yes, of course I meant local KG. The one where dispatcher gets nb2w instrument info.
But as we discussed, it could also be git repo. And for the time being we can continue to use local file, but make it persistent in the pod.

Reply:
Oh good. Maybe please modify the comment to make this clear?

Reply:
I will make it configurable, with one possibility of the local file for the moment